CHANGELOG.md

```diff
@@ -1,5 +1,44 @@
 # TigerBeetle Changelog
 
+## 2023-10-25
+
+### Features
+
+- [#1245](https://github.com/tigerbeetle/tigerbeetle/pull/1245)
+
+  Include docker images into the release.
+
+### Correctness And Performance
```

Comment:
In line with our “triple” technical values, one less syllable, and since correctness is more a superset of both safety ("bad things don't happen") and liveness/performance ("good things do happen").